src/widgets/__tests__/index.tests.ts

```diff
@@ -0,0 +1,172 @@
+import { PlacesInstance } from 'places.js';
+import * as widgets from '../';
+import { Widget } from '../../types';
+
+// @TODO: move to typedObject once this no longer needs to be polyfilled
+type Entries<TObject> = {
+  [TKey in keyof TObject]: [TKey, TObject[TKey]];
+}[keyof TObject];
+const entries = Object.entries as <TObject extends {}>(
+  yourObject: TObject
+) => Array<Entries<TObject>>;
+
+type Widgets = typeof widgets;
+type WidgetNames = keyof typeof widgets;
+
+function initiateAllWidgets(): Array<[WidgetNames, Widget]> {
```

Comment:
This has a maintenance cost and it'll be hard to detect if we miss to update this test when we introduce a new widget.

Reply:
if we miss to update this, the test will fail